chia/util/condition_tools.py

```diff
@@ -27,12 +27,7 @@ def parse_sexp_to_condition(
     if len(as_atoms) < 1:
         return Err.INVALID_CONDITION, None
     opcode = as_atoms[0]
-    try:
-        opcode = ConditionOpcode(opcode)
-    except ValueError:
-        # TODO: this remapping is bad, and should probably not happen
-        # it's simple enough to just store the opcode as a byte
-        opcode = ConditionOpcode.UNKNOWN
+    opcode = ConditionOpcode(opcode)
```

Comment:
So what happens for an unknown opcode here now? Does the exception raised get caught appropriately by all higher level callers?

Reply:
this function is only used in tests, `spend_prefarm` and `make_prefarm`, so never for unknown conditions. It would throw